### PR TITLE
RTL support

### PR DESCRIPTION
**Pull request by rudnitskih**
Open questions:
  - [ ] Tests ?
  - [x] Should we rely on config prop or global `[dir="rtl"]` attribute
  - [x] Add option in playground
  - [x] Check other possible affected areas (tooltips, subtitles)
  - [x] If we chose config prop add method to change value and use it in playground
  - [x] docs for rtl

### CODE REVIEW

File: src/modules/ui/live-indicator/live-indicator.ts [hunk 1]
```diff
@@ -99,6 +99,8 @@ export default class LiveIndicator implements ILiveIndicator {
       textMap: this._textMap,
       tooltipService: this._tooltipService,
     });
+
+    this.view.toggleActive(true);
```
Comment (bodia-uz) [inline src/modules/ui/live-indicator/live-indicator.ts]:
```suggestion
```

File: src/modules/ui/top-block/top-block.ts [hunk 2]
```diff
@@ -10,16 +10,18 @@ import {
 } from './types';
 
 import View from './top-block.view';
+import { IPlayerConfig } from '../../../core/config';
 
 interface IDependencies {
   title: ITitle;
   liveIndicator: ILiveIndicator;
+  config: IPlayerConfig;
```
Comment (bodia-uz) [inline src/modules/ui/top-block/top-block.ts]: unused?

Comment (rudnitskih) [reply src/modules/ui/top-block/top-block.ts]: nice catch

File: src/modules/ui/live-indicator/live-indicator.ts (hunk 1 above)
Comment (rudnitskih) [reply src/modules/ui/live-indicator/live-indicator.ts]: Fixed

File: src/modules/root-container/root-container.view.ts [hunk 3]
```diff
@@ -125,6 +126,10 @@ class RootContainerView extends View<IRootContainerViewStyles>
     }
   }
 
+  setRtl(rtl: boolean) {
+    this._$rootElement.setAttribute('playable-dir', rtl ? 'rtl' : 'ltr');
```
Comment (BrooklynKing) [inline src/modules/root-container/root-container.view.ts]: Can we name this attribute `data-playable-dir`?

Comment (rudnitskih) [reply src/modules/root-container/root-container.view.ts]: easy

**Comment by BrooklynKing** [comment]
WYT about 
```sass
@mixin basic($mixins...) {
  $query: ''
  @each $mixin in $mixins {
    $query: $query + $mixin
  }

  [data-playable-hook='player-container']#{$query} & {
    @content
  }
}

@function max-width-query($value) {
  @return '[data-playable-max-width~="#{$value}"]'
}

@function direction-query($direction) {
  @return if($direction, '[data-playable-dir="#{$direction}"]', '')
}

@include basic(max-width-query(550px), direction-query(ltr)) {
  margin-top: 40px;
}
```

**Pull request merged**